nvflare/app_opt/tracking/mlflow/mlflow_writer.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional
+
+from nvflare.apis.analytix import AnalyticsDataType
+from nvflare.app_common.tracking.log_writer import LogWriter
+from nvflare.app_common.tracking.tracker_types import LogWriterName
+from nvflare.app_common.widgets.streaming import ANALYTIC_EVENT_TYPE
+
+
+class MLflowWriter(LogWriter):
+    def __init__(self, event_type: str = ANALYTIC_EVENT_TYPE):
+        """MLflowWriter mimics the usage of mlflow.
+
+        For a client, replacing the import of mlflow with MLflowWriter will send log records to
+        the receiver on the server side instead of
```

Comment:
Suggest changing to the following:
```
Users can replacing the import of mlflow with MLflowWriter.
Then they will use MLflowWriter the same as they would use mlflow.
MLflowWriter will send log records to the receiver.
```

Note that the receiver does NOT necessary need to be on the server side, it could be on the client side as well.